criterion-external.cabal

```diff
@@ -21,7 +19,7 @@ executable criterion-external
                        criterion
```

Comment:
`toBenchmarkable` has only been exposed since `criterion-1.2.1.0`, so make sure to put a lower bound of `>= 1.2.1.0` here.